native/cocos/application/CocosApplication.cpp

```diff
@@ -121,7 +121,7 @@ void CocosApplication::onResume() {
 }
 
 void CocosApplication::onClose() {
-    // TODO(cc): Handling close events
+    close();
```

Comment:
In my opinion, 'close' is the act of actively invoking the shutdown, and 'onclose' is the message processed when closing, and it feels that the concepts of these two operations are confused.

Reply:
It depends on how you look at event handling, where onClose is a response to the CLOSE event, not a response to engine::close

Reply:
'onClose' is a response to the close event, but another close is called at 'onClose'?